src/libcore/ops.rs

```diff
@@ -1539,6 +1539,10 @@ impl<Idx: PartialOrd<Idx>> Range<Idx> {
 ///
 /// See the [`contains()`](#method.contains) method for its characterization.
 ///
+/// Note: No overflow checking is done for the iterator implementation; if you
+/// use an integer range and the integer overflows, it will panic in debug mode
+/// and create an endless loop in release mode.
```

Comment:
Perhaps the wording can be tweaked here slightly along the lines of:

* It will **likely** panic in debug mode, but isn't necessarily guaranteed to do so at this time.
* This will only create an endless loop for the range `0..`, right? Something like `1..` will overflow to 0 at some point which should stop iteration?